githubaction/0.1.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,28 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.1.0
+name: githubaction-webhook
+displayName: GitHub Actions Webhook Integration
+createdAt: 2021-11-19T00:00:00Z
+description: This integration shows how to invoke GitHub Action workflows leveraging Keptn's webhook service.
+logoURL: https://upload.wikimedia.org/wikipedia/commons/thumb/b/b9/Slack_Technologies_Logo.svg/2560px-Slack_Technologies_Logo.svg.png
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+  - keptn
+  - webhook
+  - github
+  - sandbox
+  - notification
+  - githubactions
+links:
+  - name: Source
+    url: https://keptn.sh/docs/0.10.x/integrations/webhooks/
+  - name: Video
+    url: https://www.youtube.com
+recommendations:
+  - url: https://artifacthub.io/packages/helm/keptn/keptn
+annotations:
+  keptn/kind: "notification"
+  keptn/org: "sandbox"
```

Comment:
Please change to `keptn/org: "keptn"`

Because the `webhook-service` lives in `keptn` organization.

Reply:
@robertjahn please change line 27 just to: `keptn/org: "keptn"`